app/controllers/posts_controller.rb

```diff
@@ -41,4 +59,13 @@ def destroy
 
     redirect_to '/'
   end
+
+  def notify
+    post_id = params[:id]
+    author_id = current_author.id
+
+    NotificationWorker.perform_in(5.minute, author_id: author_id, post_id: post_id)
```

Comment:
kiedy w workerze wprowadzimy `def perform(author_id, post_id)`
to tutaj powinno zadziałać
`NotificationWorker.perform_in(5.minute, author_id, post_id)`

Reply:
a jeśli notyfikacja ma się pojawiać bez opóźnienia to `NotificationWorker.perform_async(author_id, post_id)`